kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/ElasticsearchApiService.java

```diff
@@ -182,19 +186,38 @@ private HitsMetadata getHits(KaldbSearch.SearchResult searchResult) throws IOExc
    */
   @Get
   @Path("/:indexName/_mapping")
-  public HttpResponse mapping(@Param("indexName") Optional<String> indexName) throws IOException {
+  public HttpResponse mapping(
+      @Param("indexName") Optional<String> indexName,
+      @Param("startTimeEpochMs") Optional<Long> startTimeEpochMs,
```

Comment:
is the start/end a grafana thing? The docs link ( https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-get-mapping.html ) don't mention of a timestamp

Reply:
The start/end time is a Kaldb option for the Trino POC - https://github.com/slackhq/kaldb/pull/516#discussion_r1156199036